gpm_api/tests/conftest.py

```diff
@@ -0,0 +1,322 @@
+            "product": "2A-DPR",
+            "product_category": "radar",
+            "product_type": "RS",
+            "start_time": datetime.datetime(2020, 7, 5, 18, 33, 18),
+            "end_time": datetime.datetime(2020, 7, 5, 20, 5, 50),
+            "version": 7,
+        },
+        "ftps://arthurhouftps.pps.eosdis.nasa.gov/gpmdata/2020/07/05/radar/2A.GPM.DPR.V9-20211125.20200705-S200551-E213823.036094.V07A.HDF5": {
+            "year": 2020,
+            "month": 7,
+            "day": 5,
+            "product": "2A-DPR",
+            "product_category": "radar",
+            "product_type": "RS",
+            "start_time": datetime.datetime(2020, 7, 5, 20, 5, 51),
+            "end_time": datetime.datetime(2020, 7, 5, 21, 38, 23),
+            "version": 7,
+        },
+        # Include non-ftps folders
+        "ftp://arthurhouftps.pps.eosdis.nasa.gov/gpmdata/2020/07/05/radar/2A.GPM.DPR.V9-20211125.20200705-S213824-E231056.036095.V07A.HDF5": {
+            "year": 2020,
+            "month": 7,
+            "day": 5,
+            "product": "2A-DPR",
+            "product_category": "radar",
+            "product_type": "RS",
+            "start_time": datetime.datetime(2020, 7, 5, 21, 38, 24),
+            "end_time": datetime.datetime(2020, 7, 5, 23, 10, 56),
+            "version": 7,
+        },
+        "ftp://arthurhouftps.pps.eosdis.nasa.gov/gpmdata/2020/07/05/radar/2A.GPM.DPR.V9-20211125.20200705-S231057-E004329.036096.V07A.HDF5": {
+            "year": 2020,
+            "month": 7,
+            "day": 5,
+            "product": "2A-DPR",
+            "product_category": "radar",
+            "product_type": "RS",
+            "start_time": datetime.datetime(2020, 7, 5, 23, 10, 57),
+            "end_time": datetime.datetime(2020, 7, 6, 0, 43, 29),
+            "version": 7,
+        },
+        "ftp://arthurhouftps.pps.eosdis.nasa.gov/gpmdata/2020/07/05/radar/2A.GPM.DPR.V9-20211125.20200705-S004330-E021602.036097.V07A.HDF5": {
+            "year": 2020,
+            "month": 7,
+            "day": 5,
+            "product": "2A-DPR",
+            "product_category": "radar",
+            "product_type": "RS",
+            "start_time": datetime.datetime(2020, 7, 5, 0, 43, 30),
+            "end_time": datetime.datetime(2020, 7, 5, 2, 16, 2),
+            "version": 7,
+        },
+        "ftp://arthurhouftps.pps.eosdis.nasa.gov/gpmdata/2019/07/05/radar/2A.GPM.DPR.V9-20211125.20190705-S004330-E021602.036097.V07A.HDF5": {
+            "year": 2019,
+            "month": 7,
+            "day": 5,
+            "product": "2A-DPR",
+            "product_category": "radar",
+            "product_type": "RS",
+            "start_time": datetime.datetime(2019, 7, 5, 0, 43, 30),
+            "end_time": datetime.datetime(2019, 7, 5, 2, 16, 2),
+            "version": 7,
+        },
+        # TODO: Add more products with varying attributes ...
+    }
+
+
+@pytest.fixture
+def local_filepaths() -> List[Tuple[str, ...]]:
```

Comment:
Also here. A function can be used by maintainers to create a YAML file with the corresponding destination path elements.

Reply:
I agree